### ISSUE TITLE
Nav: clean up HTML elements and ARIA attributes

### ISSUE THREAD

**Issue by andrewrubin** [wethegit/corgi#59]
- We should be using `<ul>` instead of `<menu>`
- The `aria-expanded` attribute should be on the toggle button, not the nav list
  - Let's add `aria-controls` to the main `<nav>` itself

**Comment by andrewrubin** [comment]
This is merged to dev and will be fixed in v1.0.2
